process/peer/process.go

```diff
@@ -715,6 +738,28 @@ func (vs *validatorStatistics) getTempRating(s string) uint32 {
 	return peer.GetTempRating()
 }
 
+func (vs *validatorStatistics) updateRatingFromTempRating(pks []string) error {
+	rootHash, _ := vs.RootHash()
+	log.Trace("UpdateRatingFromTempRating before", "rootHash", rootHash)
+	for _, pk := range pks {
+		peer, err := vs.GetPeerAccount([]byte(pk))
+		if err != nil {
+			return err
+		}
+
+		tempRating := vs.getTempRating(pk)
+		rating := vs.getRating(pk)
+		log.Trace("UpdateRatingFromTempRating", "pk", []byte(pk), "rating", rating, "tempRating", tempRating)
+		err = peer.SetRatingWithJournal(vs.getTempRating(pk))
+		if err != nil {
+			return err
+		}
+	}
+	rootHash, _ = vs.RootHash()
```

Comment:
err should be treated as not critical, but logged at least

Reply:
fixed. log.warn